azurerm/internal/services/monitor/monitor_action_group_resource.go

```diff
@@ -569,6 +598,13 @@ func expandMonitorActionGroupWebHookReceiver(v []interface{}) *[]insights.Webhoo
 			ServiceURI:           utils.String(val["service_uri"].(string)),
 			UseCommonAlertSchema: utils.Bool(val["use_common_alert_schema"].(bool)),
 		}
+		if v, ok := val["aad_auth"].([]interface{}); ok && len(v) > 0 {
+			secureWebhook := v[0].(map[string]interface{})
+			receiver.UseAadAuth = utils.Bool(true)
+			receiver.ObjectID = utils.String(secureWebhook["object_id"].(string))
+			receiver.IdentifierURI = utils.String(secureWebhook["identifier_uri"].(string))
+			receiver.TenantID = utils.String(secureWebhook["tenant_id"].(string))
```

Comment:
this should default to the current tenant id if it's left unspecified e.g.

```suggestion
			if v := secureWebhook["tenant_id"].(string); v != "" {
				receiver.TenantID = utils.String(v)
			} else {
				tenantId := meta.(*clients.Client).Account.TenantID
				receiver.TenantID = utils.String(tenantId)
			}
```